.github/workflows/build_and_test.yaml

```diff
@@ -101,6 +101,7 @@ jobs:
       env:
         KIND_NODE_TAG: ${{ matrix.version }}
         IMAGE_PULL_POLICY: IfNotPresent
+        TAG: latest
```

Comment:
curious why this is needed ?
https://github.com/envoyproxy/gateway/blob/c5cd964da2f2ec1951ba72d45b54eef8246e96ee/tools/make/env.mk#L17 is the default